lib/miq_automation_engine/service_models/miq_ae_service_user.rb

```diff
@@ -3,6 +3,7 @@ class MiqAeServiceUser < MiqAeServiceModelBase
     expose :current_group,  :association => true
     expose :current_tenant, :association => true
     expose :vms,            :association => true
+    expose :miq_groups,     :association => true
```

Comment:
I don't believe so.  The only thing that might be different is that miq_groups no longer have the filters as that "entitlement" concept was promoted to an actual Entitlement model in #8102.  cc @chrisarcand